src/components/LinkCardCollection/LinkCardCollection.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Grid } from '@aws-amplify/ui-react';
+
+import LinkCard from '../LinkCard';
+
+const LinkCardCollection = () => {
+  return (
+    <Grid
+      templateColumns={{
+        base: '1fr',
+        small: '1fr 1fr',
+        large: '1fr 1fr 1fr 1fr'
+      }}
+      gap="small"
+    >
+      {/* Todo remove these added for testing purpose */}
+      <LinkCard isExternal={true} href={''} imageUrl={''}>
```

Comment:
Let's not hard code these LinkCard elements. Developers should be able to use LinkCardCollection like this:
```
<LinkCardCollection>
  <LinkCard isExternal={true} href={''} imageUrl={''}>
    {'CardData.name'}
  </LinkCard>
</LinkCardCollection>
```
So that we can provide unique links on different pages when we want. You can simplify LinkCardCollection to outputting { children } instead and LinkCardCollection just becomes a simple component to provide the responsive styling.

Reply:
Yes makes sense, added a note there, will update thanks!